src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/DependencyAnalysis/TypeMetadataNode.cs

```diff
@@ -24,11 +24,13 @@ namespace ILCompiler.DependencyAnalysis
     internal sealed class TypeMetadataNode : DependencyNodeCore<NodeFactory>
     {
         private readonly MetadataType _type;
+        private readonly bool _isMinimal;
```

Comment:
Could you please add a comment describing what it means for type metadata to be "minimal"?
And why is it OK to not include custom attributes in this case for example.